client/rhel/spacewalk-client-tools/src/firstboot-legacy-rhel5/rhn_login_gui.py

```diff
@@ -40,9 +40,9 @@
 class RhnLoginWindow(RhnRegisterFirstbootGuiWindow, rhnregGui.LoginPage):
     #You must specify a runPriority for the order in which you wish your module to run
     runPriority = 108
-    moduleName = _("Red Hat Login")
+    moduleName = _("Login")
     windowTitle = moduleName
-    shortMessage = _("Register with Red Hat Satellite")
+    shortMessage = _("Register with Uyuni")
```

Comment:
Is this string even used these days? I'm surprised this string is in the Python code but not in https://github.com/uyuni-project/uyuni/blob/master/client/rhel/spacewalk-client-tools/po/rhn-client-tools.pot

Maybe our string-extraction is not looking at this "firstboot-legay-rhel5" directory? Should it? According to the [Makefile](https://github.com/uyuni-project/uyuni/blob/master/client/rhel/spacewalk-client-tools/src/firstboot-legacy-rhel5/Makefile) in that directory, that installs "man pages for the up2date client". Do we use the up2date  client at all?

Reply:
> Is this string even used these days? 
I don't know. I didn't analyze every one string usage here, the task would take even longer.

> Do we use the up2date client at all?
AFAIK yes. up2date client _is_ the traditional stack.